src/scss/dp/overrides/components/_search.scss

```diff
@@ -149,19 +165,49 @@
       height: ($baseline * 5);
       padding-top: ($baseline / 2);
     }
+    //
+    //&--body {
+    //  height: 52px;
+    //  width: 52px;
+    //  margin-top: -2px;
+    //  float: left;
+    //}
+  }
+  &__item {
+    list-style: none;
+    padding-left: 0;
+  }
+}
 
-    &--body {
-      height: 52px;
-      width: 52px;
-      margin-top: -2px;
-      float: left;
-    }
+.search-summary {
+  width: 100% !important;
+  padding: 36px 0 46px;
+
+  .base-font {
+    font-size: 30px;
```

Comment:
should you be using rem or em as font values for the website to meet accessibility requirements?  (sorry if it is not relevant have no clue what this app is for)

Reply:
fixed